src/mrps_app.erl

```diff
@@ -4,11 +4,15 @@
 
 -export([start/2, stop/1]).
 
+-define(REGISTER, mrps_ets_register).
+
 start(_StartType, _StartArgs) ->
+    mrps_register:start(?REGISTER, []),
```

Comment:
Passing `[]` will pass zero args to `init` which is defined as `init/1`, you should do the following

```suggestion
    mrps_register:start(?REGISTER, [[]]),
```